package.json

```diff
@@ -36,6 +36,12 @@
     "versionup:major": "turbo run version --no-git-tag-version major",
     "prepare": "husky install"
   },
+  "dependencies": {
+    "@cloudscape-design/collection-hooks": "1.0.22",
```

Comment:
why are we nailing it down to a specific version ? i.e no ~ ?

Reply:
Cloudscape doesn't follow SemVer and they release breaking changes in patch version bumps. I figured this was safer.